pkg/test/runner/README.md

```diff
@@ -22,6 +22,7 @@ in `.expected`:
     used to run the test. Default: `render`.
   - `exitCode`: The expected exit code for the command. Default: 0.
   - `skip`: Runner will skip the test if `skip` is set to true. Default: false.
+  - `skipIfLocal`: Runner will skip the test if the env var IN_CI is "false" or not set. Default: false.
```

Comment:
I'd suggest to make the 2 names more consistent in terms of naming. For example, `skipIfNotInCI` with `KPT_E2E_IN_CI`.

Reply:
done!